scripts/pre_commit_linter.py

```diff
@@ -1071,36 +1119,23 @@ def _check_directive_scope(self):
             print '----------------------------------------'
         # Select JS files which need to be checked.
         files_to_check = [
-            filepath for filepath in self.all_filepaths if (
-                filepath.endswith('.js'))
-            and not any(fnmatch.fnmatch(filepath, pattern) for pattern in
-                        EXCLUDED_PATHS)]
+            filename for filename in self.all_filepaths if
```

Comment:
I think we changed this to `filepath` for accuracy in a previous PR. /cc @anubhavsinha98

Reply:
done